Procfile

```diff
@@ -0,0 +1 @@
+web: python manage.py runserver 0.0.0.0:$PORT
```

Comment:
Yes, that was the missing link. But I had to make use of heroku node.js and python build packs. After brainstorming with a couple of guys here, we realised that the app was taking too long to install in the servers. So the process never ran to completion. The build packs solved the issue because they help to install dependencies.